torchtune/modules/peft/lora.py

```diff
@@ -111,6 +118,19 @@ def adapter_params(self) -> List[str]:
         adapter_params = ["lora_a.weight", "lora_b.weight"]
         return adapter_params
 
+    def dora_init(self) -> None:
```

Comment:
Hello @rohan-varma and @ebsmothers, I am wondering if there is a more efficient approach to the problem I am facing. I have followed the PEFT (Parameter Efficient Fine-Tuning) method but I am encountering an issue with the initialization of the self.m (magnitude vector) in torch tune because I require the weights. Currently, I am using the flag (self.dora_initialized) to address this issue, but I believe this may not be the best solution. I would appreciate any suggestions you may have.